src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -1,11 +1,22 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.Table;
 import java.util.ArrayList;
 import java.util.List;
 
+@Entity
+@Table(name = "movies")
 public class Movie implements Cloneable {
```

Comment:
Implementing Cloneable and providing a clone method is not typical for entities in Hibernate. Entities are usually managed by the persistence context, and cloning them can lead to unexpected behavior, especially with related managed entities. Consider removing the Cloneable interface and the clone method unless there is a specific requirement for this in your project that cannot be handled in a more idiomatic way.